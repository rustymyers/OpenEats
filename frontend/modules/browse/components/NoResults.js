import React from 'react'
import PropTypes from 'prop-types'
import {
    injectIntl,
    defineMessages
} from 'react-intl';

const NoResults = ({intl}) => {
  const messages = defineMessages({
    no_results: {
      id: 'browse.no_results',
      description: 'No results header',
      defaultMessage: 'Sorry, there are no results for your search.',
    }
  });

  return (
    <div className="row">
      <div className="col-xs-12">
        <div id="browse" className="row">
          <div className="spinner">
            <h3 className="no-results">{ intl.formatMessage(messages.no_results) }</h3>
          </div>
        </div>
      </div>
    </div>
  )
};

NoResults.propTypes = {
  intl: PropTypes.object
};

export default injectIntl(NoResults);