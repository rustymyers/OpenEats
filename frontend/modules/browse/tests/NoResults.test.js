import React from 'react';
import NoResults from '../components/NoResults';
import createComponentWithIntl from '../../../jest_mocks/createComponentWithIntl';

test('NoResults component test', () => {
  const component = createComponentWithIntl(
    <NoResults/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
