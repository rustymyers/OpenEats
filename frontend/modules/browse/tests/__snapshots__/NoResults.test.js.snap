// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component test 1`] = `
<div
  className="row"
>
  <div
    className="col-xs-12"
  >
    <div
      className="row"
      id="browse"
    >
      <div
        className="spinner"
      >
        <h3
          className="no-results"
        >
          Sorry, there are no results for your search.
        </h3>
      </div>
    </div>
  </div>
</div>
`;
